examples/asset-explorer/tsconfig.json

```diff
@@ -0,0 +1,119 @@
+{
+  "compilerOptions": {
+    "target": "ESNext",
+    "lib": [
+      "dom",
+      "dom.iterable",
+      "esnext"
+    ],
+    "allowJs": true,
+    "skipLibCheck": true,
+    "strict": true,
+    "forceConsistentCasingInFileNames": true,
+    "noEmit": true,
+    "esModuleInterop": true,
+    "module": "esnext",
+    "moduleResolution": "node",
+    "resolveJsonModule": true,
+    "isolatedModules": true,
+    "jsx": "preserve",
+    "baseUrl": "./",
+    "paths": {
+      "actions": [
+        "src/actions"
+      ],
+      "components/*": [
+        "src/components/*"
+      ],
+      "components": [
+        "src/components"
+      ],
+      "const/*": [
+        "src/const/*"
+      ],
+      "const": [
+        "src/const"
+      ],
+      "hooks/*": [
+        "src/hooks/*"
+      ],
+      "hooks": [
+        "src/hooks"
+      ],
+      "utils/*": [
+        "src/utils/*"
+      ],
+      "utils": [
+        "src/utils"
+      ],
+      "types/*": [
+        "src/types/*"
+      ],
+      "types": [
+        "src/types"
+      ],
+      "definitions/*": [
+        "src/definitions/*"
+      ],
+      "definitions": [
+        "src/definitions"
+      ],
+      "interfaces": [
+        "src/interfaces"
+      ],
+      "interfaces/*": [
+        "src/interfaces/*"
+      ],
+      "pages/*": [
+        "src/pages/*"
+      ],
+      "reducers": [
+        "src/redux/reducers"
+      ],
+      "reducers/*": [
+        "src/redux/reducers/*"
+      ],
+      "redux": [
+        "src/redux"
+      ],
+      "redux/*": [
```

Comment:
looks like redux is not used at all, but maybe we could delete all custom resolvers to keep the example simple